src/app/services/contracts.service.ts

```diff
@@ -0,0 +1,156 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {NGXLogger} from 'ngx-logger';
+import {Observable} from 'rxjs';
+import {environment} from '../../environments/environment';
+import {catchError, filter, map} from 'rxjs/operators';
+import {SnackService} from './snack.service';
+import {StaticValues} from '../static/static-values';
+import {Addresses} from '../static/addresses';
+
+interface Result<T> {
+    data: T | Array<T>;
+}
+
+// eslint-disable-next-line
+export interface IsContract{}
+
+export class Contract implements IsContract {
+    id: number;
+    address: string;
+    name: string;
+    prettyName() {
```

Comment:
I disabled element ordering in eslint but please, put functions after variables

Reply:
👍